src/App.vue

```diff
@@ -0,0 +1,30 @@
+<template>
+  <div id="app">
+    <app-header></app-header>
+    <router-view></router-view>
+  </div>
+</template>
+<script>
```

Comment:
Space in between closing template tag and opening script tag. 
There are other instances as well in this PR.